src/components/pages/advanced.tsx

```diff
@@ -0,0 +1,70 @@
+import robot1 from '../../assets/robots/pg6/robot1pg6.svg';
+import robot2 from '../../assets/robots/pg6/robot2pg6.svg';
+import robot3 from '../../assets/robots/pg6/robot3pg6.svg';
+import robot4 from '../../assets/robots/pg6/robot4pg6.svg';
+import {lineOfCode, CodeFormat}  from  '../shared/codeFormat';
+import {Color} from '../shared/constants';
+
+function Advanced(): JSX.Element {
+
+  const code: lineOfCode[] = [
+    [
+      { ['print']: Color.Orange },
+      { ['(num_of_antennas']: Color.White },
+      { [' > ']: Color.Orange },
+      { ['1']: Color.Blue },
+      { [' or ']: Color.Orange },
+      { ['has_mouth)']: Color.White },
+    ],
+    [
+      { ['print']: Color.Orange },
+      { ['(color']: Color.White },
+      { [' == ']: Color.Orange },
+      { ['"red"']: Color.White },
+      { [' and ']: Color.Orange },
+      { ['color']: Color.White },
+      { [' != ']: Color.Orange },
+      { ['"blue")']: Color.White },
+    ],
+    [
+      { ['print']: Color.Orange },
+      { ['(']: Color.White },
+      { ['not ']: Color.Orange },
+      { ['head_shape']: Color.White },
+      { [' == ']: Color.Orange },
+      { ['"circle")']: Color.White },
+    ],
+  ];
+
+  return (
+    <div className="frame">
+      <div id="landing" className='robot-layout'>
```

Comment:
Can we take out` id="landing"`? This goes for all the other pages that have it too.